enhancements/builds/remove-jenkins-playload.md

```diff
@@ -0,0 +1,195 @@
+---
+title: remove-jenkins-payload
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@gabemontero"
+  - "@akram"
+approvers:
+  - "@bparees"
+  - "@sbose78"
+  - "@derekwaynecarr"
+creation-date: 2021-07-20
+last-updated: 2021-07-20
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Remove Jenkins from the OCP Payload
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This proposal aims to remove Jenkins images from the OCP payload and replace them with images delivered via CPaaS.
+
+## Motivation
+
+Distributing and supporting Jenkins directly has always been challenging for OpenShift.
+Due to OpenShift's tight coupling to Jenkins via the Jenkins Pipeline build strategy,
+OpenShift 4 has included a Jenkins distribution as well as a default set of agent images used to run Jenkins pipelines.
+This meant that any bug fixes or upgrades to Jenkins needed to go through the often arduous backport process to get delivered to customers.
+
+Jenkins itself is not native to Kubernetes and does not use a Java runtime that has been optimized for cloud-native environments (i.e. Quarkus).
```

Comment:
if you'd like to include some sort of reference to Jenkins and Quarkus @adambkaplan, there minimally is https://www.jenkins.io/sigs/platform/